UnitedSets/UnitedSets.csproj

```diff
@@ -83,7 +84,6 @@
 		<ProjectReference Include="..\Cube.UI\Cube.UI.csproj" />
 		<ProjectReference Include="..\EasyXAMLTools\EasyCSharp\EasyCSharp\EasyCSharp.csproj"></ProjectReference>
 		<ProjectReference Include="..\EasyXAMLTools\EasyCSharp\EasyCSharp.Generator\EasyCSharp.Generator.csproj" OutputItemType="Analyzer" ReferenceOutputAssembly="false" />
-		<ProjectReference Include="..\EasyXAMLTools\EasyXAMLTools.Generator\EasyXAMLTools.Generator.csproj" OutputItemType="Analyzer" ReferenceOutputAssembly="false" />
```

Comment:
Hmm... why are we removing the generator?

Reply:
Mistake, i think during a master merge as I can't even find the commit that had done it, readd and repushed.